tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/CelebornFetchFailureSuite.scala

```diff
@@ -297,4 +300,165 @@ class CelebornFetchFailureSuite extends AnyFunSuite
       sparkSession.stop()
     }
   }
+
+  test(s"celeborn spark integration test - resubmit an unordered barrier stage with throwsFetchFailure enabled") {
+    val sparkConf = new SparkConf().setAppName("rss-demo").setMaster("local[2]")
+    val sparkSession = SparkSession.builder()
+      .config(updateSparkConf(sparkConf, ShuffleMode.HASH))
+      .config("spark.sql.shuffle.partitions", 2)
+      .config("spark.celeborn.shuffle.forceFallback.partition.enabled", false)
+      .config("spark.celeborn.shuffle.enabled", "true")
```

Comment:
nit: seems this config is not defined by Celeborn?

Reply:
Indeed !
This was copy + paste of existing tests in the suite ... I should have paid more attention :-(

Let me remove this config from rest of the tests in this file as well. Thanks for catching it !

Reply:
Removed.